specification.md

```diff
@@ -2464,6 +2484,50 @@ to be used as payload of the event referenced by `triggerEventRef`. If it is of
 The `contextAttributes` property allows you to add one or more [extension context attributes](https://github.com/cloudevents/spec/blob/master/spec.md#extension-context-attributes)
 to the trigger/produced event. 
 
+#### SubFlowRef Definition
+
+| Parameter | Description | Type | Required |
+| --- | --- | --- | --- |
+| waitForCompletion | If workflow execution must wait for sub-workflow to finish before continuing | boolean | yes |
+| workflowId |Sub-workflow unique id | boolean | no |
+
+<details><summary><strong>Click to view example definition</strong></summary>
+<p>
+
+<table>
+<tr>
+    <th>JSON</th>
+    <th>YAML</th>
+</tr>
+<tr>
+<td valign="top">
+
+```json
+{
+    "workflowId": "handleApprovedVisaWorkflowID"
+}
+```
+
+</td>
+<td valign="top">
+
+```yaml
+workflowId: handleApprovedVisaWorkflowID
+```
+
+</td>
+</tr>
+</table>
+
+</details>
+
+The `waitForCompletion` property defines if the SubFlow action should wait until the referenced reusable workflow
+has completed its execution. If it's set to "true" (default value), SubFlow action execution must wait until the referenced workflow has completed its execution.
+In this case the workflow data output of the referenced workflow will be used as the result data of the action.
+If it is set to "false" the parent workflow can continue its execution as soon as the referenced sub-workflow 
```

Comment:
From the meeting discussion, this section here talks about asynch sub-flow & how the parent workflow should behave in that case.